std/typecons.d

```diff
@@ -2969,7 +2969,13 @@ Returns:
     }
 
     /// ditto
-    @property get(U)(inout(U) fallback) inout @safe pure nothrow
+    @property inout(T) get()(inout(T) fallback) inout @safe pure nothrow
+    {
+        return isNull ? fallback : _value.payload;
+    }
+
+    /// ditto
```

Comment:
You might want to document its special (and IMHO, surprising) semantic.

Reply:
documented?